common/app/views/fragments/basher.scala.html

```diff
@@ -0,0 +1,23 @@
+@(id: String)(implicit request: RequestHeader)
+
+@import views.support.Basher
+
+@Basher.fromId(id).map { id =>
+    <div class="facia-basher" data-link-name="basher">
+        @conf.Static.css.inline(id.toString).map { css =>
+            @if(play.Play.isDev()) {
+                <link rel="stylesheet" type="text/css" href="@Static("stylesheets/inline/basher." + id.toString + ".css")" />
```

Comment:
because `css` in this instance is the actual css, not a path.

I've moved the map to just the inlining for better readability.